eth/v1/BUILD.bazel

```diff
@@ -30,38 +73,50 @@ ssz_gen_marshal(
     ],
 )
 
-proto_library(
-    name = "v1_proto",
-    srcs = [
-        "attestation.proto",
-        "beacon_block.proto",
-        "beacon_chain_service.proto",
-        "beacon_debug_service.proto",
-        "beacon_state.proto",
-        "node.proto",
-        "validator.proto",
-        "validator_service.proto",
+go_proto_library(
+    name = "go_proto",
+    compilers = [
+        "@com_github_prysmaticlabs_protoc_gen_go_cast//:go_cast_grpc",
     ],
+    importpath = "github.com/prysmaticlabs/ethereumapis/eth/v1",
+    proto = ":proto",
     visibility = ["//visibility:public"],
     deps = [
-        "@com_google_protobuf//:empty_proto",
-        "@com_google_protobuf//:timestamp_proto",
-        "@go_googleapis//google/api:annotations_proto",
-        "@gogo_special_proto//github.com/gogo/protobuf/gogoproto",  # keep
+        "//eth/ext:go_default_library",
+        "@com_github_golang_protobuf//descriptor:go_default_library",
+        "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_golang_protobuf//protoc-gen-go/descriptor:go_default_library",
+        "@com_github_golang_protobuf//ptypes/empty:go_default_library",
+        "@com_github_golang_protobuf//ptypes/timestamp:go_default_library",
+        "@com_github_prysmaticlabs_go_bitfield//:go_default_library",  # keep
+        "@com_github_prysmaticlabs_eth2_types//:go_default_library",
+        "@go_googleapis//google/api:annotations_go_proto",
+        "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
+        "@org_golang_google_protobuf//runtime/protoimpl:go_default_library",
     ],
 )
 
-go_proto_library(
-    name = "v1_go_proto",
-    compilers = ["@io_bazel_rules_go//proto:gogofast_grpc"],
+go_library(
+    name = "go_grpc_gateway_library",
+    srcs = glob(["*.pb.gw.go"]),
```

Comment:
I did this because of the types.Epoch/Slot changes that were made as my plugin doesnt touch the gRPC files for casting. Victor will look into some more maintainable solution than manually modifying the gateway files into a compiling state.